src/modules/ekf2/EKF/airspeed_fusion.cpp

```diff
@@ -61,18 +61,27 @@ void Ekf::controlAirDataFusion(const imuSample &imu_delayed)
 		_control_status.flags.wind = false;
 	}
 
+	// clear yaw estimator airspeed (updated later with true airspeed if airspeed fusion is active)
```

Comment:
Okay, but at some point I think we should extract the yaw estimator completely and only run a single instance even when running multiple ekfs so we shouldn't make it more deeply integrated with other ekf2 control logic.

Reply:
I'm not opposed, I actually did an initial version of this in a branch years ago when the yaw estimator was first introduced.

Where I stopped was thinking there's still potential value in having a yaw estimator for each IMU (given each yaw estimator is not that expensive) and possibly each GPS. Should the yaw estimator itself run with instances for each IMU (x GPS) or do you think it's good enough on only the current primary?